File: C#/프로그래머스/0/181919. 콜라츠 수열 만들기/콜라츠 수열 만들기.cs
using System;
using System.Collections;

public class Solution {
    public ArrayList solution(int n) {
        ArrayList answer = new ArrayList();
        answer.Add(n);
        while(n!=1){
            if(n%2==0){
                n/=2;
            }
            else{
                n*=3;
                n++;
            }
            answer.Add(n);
        }
        return answer;
    }
}

File: Java/프로그래머스/0/181919. 콜라츠 수열 만들기/콜라츠 수열 만들기.java
import java.util.ArrayList;

class Solution {
    public ArrayList<Integer> solution(int n) {
        ArrayList<Integer> answer = new ArrayList<>();
        answer.add(n);
        while(n!=1){
            if (n%2==0){
                n/=2;
            }
            else{
                n=n*3+1;
            }
            answer.add(n);
        }
        return answer;
    }
}